Refactor instance test

keiichi-hikita opened the pull request:
Now network suitable for ECL2.0 functionality
Is supported, so continue to run the remaining tests.
Then all tests are correctly passed.

rintooo commented:
Let me write a comment.
Comment detail is the same with the one written in eclcloud.

Are these schemas supported in ECL2.0??
```
"network": &schema.Schema{
    Elem: &schema.Resource{
	Schema: map[string]*schema.Schema{
            "name": &schema.Schema{}
            "port": &schema.Schema{}
            "mac": &schema.Schema{}
        }
    }
}
```
```
"access_ip_v4": &schema.Schema{}
```
```

keiichi-hikita commented:
Thanks for your review.

#### About network schema

Let me check about supported schema around network.
I'll implement another Acc test and check them.

#### About stop_before destryo.

stop_before_destroy is not looked as kind of schema.
Smae like "power_state", user can control power state of instance when user destroy this instance.
If this parameter is set as TRUE, prior to destroy operatino from Terraform, instance will be shutdown.
And this parameter is tested through ACC test already. So we can expose this parameter to customer.

keiichi-hikita commented:
I added two of new test cases regarding your comments.

> "port": &schema.Schema{}
> "mac": &schema.Schema{}

Those two parameters are tested by following test function.
`TestAccComputeV2InstanceConnectToCreatedPort`

`mac` is computed only parameter so user can not set, but it will be saved into state as a result of port connection.

> "name": &schema.Schema{}

Same as the above cases, it is tested by `TestAccComputeV2InstanceConnectToNetworkByName`

Please have a look at above two test cases o/

In this Pull Request, implicit port problem(Subnet deletion becomes conflict status) is also cleared by adding retry logic into subnet resource.
I passed all of subnet relevant tests like followings.

```
TESTNAME=NetworkV2Subnet
make testacc TEST=./ecl TESTARGS="-run=$TESTNAME -count=1"
```
And instance tests as well.

I've fixed my PR, please review again.
